src/main/java/com/box/sdk/BoxMetadataCascadePolicy.java

```diff
@@ -145,7 +145,7 @@ public void forceApply(String conflictResolution) {
         JsonObject requestJSON = new JsonObject()
                 .add("conflict_resolution", conflictResolution);
         request.setBody(requestJSON.toString());
-        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        BoxAPIResponse response = request.send();
```

Comment:
If you're not using the result anyway, why have it at all?  You could just do `request.send()` and throw away the result.